server/api/orders.js

```diff
@@ -1,75 +1,77 @@
 const router = require('express').Router()
-const {Transactions} = require('../db/models')
+const {Orders} = require('../db/models')
 //const {isAminMiddleware} = require('../admin.middleware')
 module.exports = router
 
-//GET all transactions for all users (ADMIN)
+//All ADMIN Routes
+
+//GET all orders for all users
 router.get('/', async (req, res, next) => {
   try {
-    const allTransactions = await Transactions.findAll()
-    res.json(allTransactions)
+    const allOrders = await Orders.findAll()
+    res.status(200).json(allOrders)
   } catch (err) {
     next(err)
   }
 })
 
-//GET all transactions for a user
+//GET all orders for a single user
 router.get('/:userid', async (req, res, next) => {
   try {
     const id = req.params.id
-    const transactions = await Transactions.findAll({
+    const orders = await Orders.findAll({
       where: {
         userId: id
       }
     })
-    if (!transactions) return res.sendStatus(404)
-    res.json(transactions)
+    if (!orders) return res.sendStatus(404)
```

Comment:
order is not a primitive data type; So this condition will always evaliuate to true